### PR TITLE
Pokemon Emerald: Exclude sacred ash post champion

### PR DESCRIPTION
**Pull request by Zunawe**
## What is this fixing or adding?

Excludes the hidden item in Navel Rock if event tickets are not randomized and goal is champion, since the event ticket that allows you to reach Navel Rock is locked behind champion.

## How was this tested?

Checked the location progress type of the location after `create_regions` to make sure it was excluded. Also ran the code when hidden items were turned off to make sure it didn't crash from the location not existing.

### CODE REVIEW

File: worlds/pokemon_emerald/__init__.py [hunk 1]
```diff
@@ -288,6 +288,12 @@ def exclude_locations(location_names: List[str]):
                     "Safari Zone SE - Hidden Item in South Grass 2",
                     "Safari Zone SE - Item in Grass",
                 ])
+
+            # Sacred ash is on Navel Rock, which is locked behind the event tickets
+            if not self.options.event_tickets.value:
```
Comment (Exempt-Medic) [inline worlds/pokemon_emerald/__init__.py:293]: You shouldn't need the `.value` here and don't use it in the same constructions elsewhere in this file
```suggestion
            if not self.options.event_tickets:
```

**Review by Exempt-Medic — APPROVED** [review]
One small comment, but the changes LGTM, match my understanding of the game. Tested (with the commented change) the relevant option values hit the code path and that it actually worked through print debugging and looking at the location's progress type

File: worlds/pokemon_emerald/__init__.py (hunk 1 above)
Comment (nicholassaylor) [reply worlds/pokemon_emerald/__init__.py:293]: I agree with this statement

**Review by nicholassaylor — APPROVED** [review]
LGTM, same comment that Medic had

**Pull request merged**